core/deployment/src/main/java/io/quarkus/deployment/ide/IdeUtil.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.deployment.ide;
+
+import java.util.Locale;
+
+final class IdeUtil {
+
+    private IdeUtil() {
+    }
+
+    static boolean isWindows() {
+        String os = System.getProperty("os.name");
+        return os.toLowerCase(Locale.ENGLISH).startsWith("windows");
+    }
+}
```

Comment:
`isWindows()` is currently used only in the "ide" package, but it seems not be dependent on only "ide". Isn't it better to change the class name to be more general? like 'SystemUtil' or 'PlatfromUtil'. And since the package '../deploy/util' already exists, wouldn't this class be better to be in it?

Reply:
Yes, if the need arises, I'll move it